aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -355,13 +423,16 @@ const CancellationLayer = ({ onSetOpen }) => {
           pad={{ top: 'medium', bottom: 'small' }}
```

Comment:
This is adding extra padding (both above and below) when compared with Figma.

Suggest removing padding here and adjusting line 406 to be a medium gap.

Reply:
Thanks, adjusted.